tools/coco_limit_image_size.py

```diff
@@ -56,12 +62,23 @@ def main():
             image = cv2.resize(image, new_size)
             resized += 1
 
+            for i in images_idx_to_annotations_idx[image_info['id']]:
+                content['annotations'][i]['bbox'] = [x / ratio for x in
+                                                     content['annotations'][i]['bbox']]
+                if content['annotations'][i]['segmentation']:
+                    content['annotations'][i]['segmentation'] = [
+                        [x / ratio for x in c] for c in content['annotations'][i]['segmentation']
+                    ]
```

Comment:
Have you tried running this on MS COCO annotation files?

Reply:
good catch, fixed